scipy/sparse/tests/test_base.py

```diff
@@ -3921,7 +3922,11 @@ def test_dot(self):
         B = lil_matrix((10,10), dtype=np.complex)
         B[0,3] = 10
         B[5,6] = 20j
-        assert_array_equal(A @ A.T, (B * B.T).todense())
+
+        # TODO: properly handle this assertion on ppc64le
+        if platform.machine() != 'ppc64le':
```

Comment:
This testing module is a bit messy--this is the only case where I didn't use pytest to directly xfail.